README.md

```diff
@@ -8,7 +8,7 @@ Python version of the TerminusDB client - for TerminusDB API and WOQLpy
 
 ## Requirements
 - [TerminusDB 2.0.5](https://github.com/terminusdb/terminusdb-server)
-- Python >= 3.6
+- [Python 3.6](https://www.python.org/downloads/release/python-360)
```

Comment:
As we are compatible with all the Python versions since 3.6 so we have to keep the `>=` to indicate that people using 3.7 and 3.8 are still compatible. For the link, maybe we can link to the latest version of Python (as it is recommended)

Reply:
ok i will make it as >=

Reply:
@Cheukting Which version should i give  Python >= 3.8 is this ok